### DISCUSSION TITLE
求大佬帮忙

### DISCUSSION THREAD

**Discussion by RiseToTop** [umijs/qiankun#2875] [Q&A]
我的是umi和antd pro作为的主应用项目，一个vue3作为子应用
  {
    path: '/optimize/*',
    microApp: 'optimize-front',
    microAppProps: {
      autoSetLoading: true,
      autoCaptureError: true,
    },
    footerRender: false,
  },
  
  我配置了autoSetLoading但那个加载图标子应用加载出来了也一直转圈，有什么办法关闭吗

**Comment by Zuojiangtao** [comment]
你的加载图标是父应用的，应该在子应用加载后主动关闭。可以在父应用写个关闭方法传给子应用，在子应用mount钩子函数调用。
